packages/js/src/utils/buildable-libs-utils.ts

```diff
@@ -194,18 +194,23 @@ function collectDependencies(
 }
 
 function readTsConfigWithRemappedPaths(
-  tsConfig: string,
-  generatedTsConfigPath: string,
-  dependencies: DependentBuildableProjectNode[]
+  originalTsconfigPath: string,
+  generatedTsconfigPath: string,
+  dependencies: DependentBuildableProjectNode[],
+  workspaceRoot: string
 ) {
   const generatedTsConfig: any = { compilerOptions: {} };
-  const dirnameTsConfig = dirname(generatedTsConfigPath);
-  const relativeTsconfig = isAbsolute(dirnameTsConfig)
-    ? relative(workspaceRoot, dirnameTsConfig)
-    : dirnameTsConfig;
-  generatedTsConfig.extends = relative(relativeTsconfig, tsConfig);
+  const normalizedTsConfig = resolve(workspaceRoot, originalTsconfigPath);
```

Comment:
With Crystal plugins (such as NxWebpackAppPlugin, and similar for Vite) which set up buildable libraries, will this always be relative to workspace root?

Reply:
The utility expects an absolute path or a path relative to the workspace root. Crystal plugins or any other consumer of the helper need to ensure the path is either absolute or relative to the workspace root.